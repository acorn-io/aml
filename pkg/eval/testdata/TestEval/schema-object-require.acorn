define Foo: {
	a: {
		b?: "x"
	}
}

Foo(
{}
)