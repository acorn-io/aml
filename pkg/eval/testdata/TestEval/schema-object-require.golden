{
  "Foo": {
    "type": "object",
    "properties": {
      "a": {
        "$ref": "#/$defs/Foo.a"
      }
    },
    "defs": {
      "Foo.a": {
        "type": "object",
        "properties": {
          "b": {
            "type": "string"
          }
        }
      }
    }
  },
  "a": {}
}